### ISSUE TITLE
[Adaptive Cards - Activity Update]: Talkback is not announcing the role of the image while navigating on the 'Activity Card' screen.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards-Mobile#375]
### Target Platforms

Android

### SDK Version

1.0.1(2)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS: Android 14
Device: S24
Screen Reader: Talkback
AdaptiveCard Android App version: 1.0.1(2)

**Repro Steps:**
1. Launch the Adaptive cards app on Android.
2. Turn on Talkback.
3. Navigate to 'Import' control and select it.
4. Navigate to 'Sample cards' and select it.
5. Navigate to available dropdown under 'Sample cards'.
6. Navigate to available dropdown under 'Sample cards' and select it.
7. Navigate all the controls under dropdown and select 'Activity Update'.
8. Navigate to the screen using left/right swipe gesture and observe the issue.

**Actual Result:**
While navigating the 'Activity Update' screen using swipe gesture, Talkback is not announcing the role as 'image' for the 'Matt Hidinger'. Currently it is announcing as **'Matt Hidinger.'**
 
**Expected Result:**
Screen Reader should announce the role as 'Image' for the matt Hidinger image. When the focus lands on it then it should announce as **'Matt Hidinger image or graphic.'**

**User Impact:**
It will impact the screen reader user as When TalkBack doesn’t announce the role or purpose of an image, users who are blind or visually impaired are left without crucial information. They won’t know if the image is decorative, informative, a button, or a link.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/name-role-value

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

https://github.com/user-attachments/assets/5efbede1-39d8-4f16-91ea-a47c61ef0951

![Image](https://github.com/user-attachments/assets/7d9d398d-7f05-4804-ace9-d63085c0b427)

### Card JSON

```JSON
NA
```
